README.md

```diff
@@ -59,8 +59,17 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 }
 ```
 
+
+SSH Authentication
+-----------------
+Ungit does run git commands in background and authentication will be asked continously when working with private repos.  Easiest way to prevent this is to set up ssh authorization and cache it.
```

Comment:
1. Fix typo.
2. "authorization" -> "authentication" (see above
```suggestion
Ungit does run git commands in background and authentication will be asked continuously when working with private repos.  Easiest way to prevent this is to set up ssh authentication and cache it.
```